lib/recurly/subscription.rb

```diff
@@ -114,6 +114,11 @@ def subscription_add_ons= subscription_add_ons
     end
     alias add_ons= subscription_add_ons=
 
+    def pending_subscription
+      sub = self[:pending_subscription]
+      sub.tap {|e| e.currency = currency} if sub.is_a? Subscription
```

Comment:
done in that fashion, `subscription.pending_subscription` would only return the currency attribute for the pending_subscription